src/frontend/src/pages/GanttPage/GanttChartSection.tsx

```diff
@@ -35,14 +38,50 @@ const GanttChartSection = ({ start, end, tasks, isEditMode, saveChanges, onExpan
   }, [isEditMode]);
 
   const displayEvents = applyChangesToEvents(tasks, eventChanges);
+  const projects = displayEvents.filter((event) => !event.project);
+
+  const toggleWorkPackages = (projectId: string) => {
+    setShowWorkPackagesList((prevState) => ({
+      ...prevState,
+      [projectId]: !prevState[projectId]
+    }));
+  };
 
   return tasks.length > 0 ? (
     <Box sx={{ width: 'fit-content' }}>
       {/* Data display: reset list of events every time eventChanges list changes using key */}
       <Box sx={{ mt: '1rem', width: 'fit-content' }} key={eventChanges.length}>
-        {displayEvents.map((event) => {
+        {projects.map((project, index) => {
           return (
-            <GanttTaskBar key={event.id} days={days} event={event} isEditMode={isEditMode} createChange={createChange} />
+            <>
+              <Box display="flex" alignItems="flex-start">
+                <Box position="fixed" zIndex="2">
+                  <IconButton onClick={() => toggleWorkPackages(project.id)}>
```

Comment:
Could this get a black background so it doesn't overlap with the text when scrolling?

Reply:
Oh sorry I meant like matching the background of the page. Ideally it would be like fully that color all the way to the left to have the bars show not underneath. Realizing this would mean shifting the timeline or moving the buttons all the way to the side of the screen which is prob easier said than done so just take a crack at it